### DISCUSSION TITLE
Primeng treetable frozen column at end of table

### DISCUSSION THREAD

**Discussion by ShilpaMore-27** [primefaces/community#524] [PrimeNG]
Is there any way to have frozen columns at the end of table in Primeng treetable control? I see that it places the frozen column only at the start.

**Comment by cetincakiroglu** [comment]
TreeTable doesn't support frozen columns at the end of the table. It replaces the frozen column at the beginning only.

**Comment by scheelu** [comment]
Hi, I'm also interested in this feature

**Comment by cetincakiroglu** [comment]
Hi,

We'll wait for community upvotes to implement this as a feature.
